res/themes/dark/css/_dark.scss

```diff
@@ -184,6 +184,9 @@ $visual-bell-bg-color: #800;
 
 $room-warning-bg-color: $header-panel-bg-color;
 
+$authpage-body-bg-color: $background;
+$authpage-primary-color: $text-primary-color;
```

Comment:
I think `$primary-content` could be better here or one of the colors at the top of the file